initial/contract-rest-client/src/test/java/hello/ContractRestClientApplicationTest.java

```diff
@@ -0,0 +1,33 @@
+package hello;
+
+import org.assertj.core.api.BDDAssertions;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.stubrunner.junit.StubRunnerRule;
+import org.springframework.cloud.contract.stubrunner.spring.AutoConfigureStubRunner;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.RestTemplate;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureStubRunner(
+		ids = "com.example:contract-rest-service:0.0.1-SNAPSHOT:stubs:8100",
+		workOffline = true
+)
+public class ContractRestClientApplicationTest {
+
+	@Test
+	public void get_person_from_service_contract() {
```

Comment:
I'd go with the same syntax as presented previously

Reply:
fixed.